README.md

```diff
@@ -25,16 +25,11 @@ $ export MESH_NAME=my_mesh_name
 $ export ENABLE_STATS_TAGS=true
 ```
 
-(Optional) If enabled, Envoy will emit DogStatsD metrics to 127.0.0.1:8125, where it expects to find a statsd receiver. This could be either a Datadog sidecar, or something like [statsd_exporter](https://github.com/prometheus/statsd_exporter) (see below).
+(Optional) If enabled, Envoy will emit DogStatsD metrics to 127.0.0.1:8125, where it expects to find a statsd receiver. 
```

Comment:
So I looked at the docs, and the appmesh envoy accepts an `ENABLE_ENVOY_DOG_STATSD` env var.   Is that configurable by the injector after this change?

Reply:
Yes. The `--enable-statsd` flag (which isn't removed by this PR) defines whether or not `ENABLE_ENVOY_DOG_STATSD` is set on the envoy container. See https://github.com/aws/aws-app-mesh-inject/blob/c8ac4885b663afe1fd6a8e31c73cdbcd130dc1b1/pkg/patch/sidecar.go#L51-L55